QuickshaReTests/QRPictoreGeneratorTests.swift

```diff
@@ -19,7 +19,7 @@ class QRPictoreGeneratorTests: XCTestCase {
         let picture = generator.qrPicture(for: message)
         
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: QuickshaReTests/QRPictoreGeneratorTests.swift;
  Line: 20;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)